python/lsst/analysis/tools/analysisMetrics/fluxMetrics.py

```diff
@@ -0,0 +1,40 @@
+from lsst.analysis.tools.actions.scalar import MeanAction, MedianAction
+from lsst.analysis.tools.interfaces import AnalysisMetric
+
+
+class FluxMeanMetric(AnalysisMetric):
+    """Create a metric of the mean psf, aperture, and total
+    flux from diaSources.
+    """
+
+    def setDefaults(self):
+        super().setDefaults()
+
+        self.process.calculateActions.psFluxMeanMetric = MeanAction(vectorKey="psFlux")
+        self.process.calculateActions.apFluxMeanMetric = MeanAction(vectorKey="apFlux")
+        self.process.calculateActions.totFluxMeanMetric = MeanAction(vectorKey="totFlux")
+
+        self.produce.units = {
+            "psFluxMeanMetric": "flx",
+            "apFluxMeanMetric": "flx",
+            "totFluxMeanMetric": "flx",
+        }
+
+
+class FluxMedianMetric(AnalysisMetric):
```

Comment:
Is there a reason you chose to set these up this way instead of having something like a "central tendency" metric? It does not really matter one way or the other for performance reasons, it just adds a second layer to the key in the final metrics output (which you may have wanted) I was just curious.

Reply:
Now that you have suggested it, central tendency would probably be a more useful metric. I went this route with the different median and mean metrics since, in our brainstorming, we wanted them as separate things as different people mentioned wanting the mean flux and the median flux separately. I might go back and discuss with folks if it would be better if they were separated out or if they should be combined, as I'm not sure if its important to keep them separate or not.